src/models/section.ts

```diff
@@ -10,7 +10,7 @@ export enum SectionType {
   didYouKnow = "didYouKnow"
 }
 
-const sectionInfo = {
+export const sectionInfo = {
```

Comment:
Hmm I know there were strategies in Java for annotating things as only exported for testing, is there a similar approach that could help here?

Reply:
Not that I'm aware of. Did you have something in mind?